lib/shinq/configuration.rb

```diff
@@ -27,6 +27,14 @@ def initialize(opts)
       end
     end
 
+    def constantize_worker_class
```

Comment:
I think better than this way remove worker_class from accessors, and create same name method(worker_class).
Then `launcher.rb`'s patch will be unnecessary.